src/Microsoft.DotNet.Darc/src/Darc/Helpers/UxHelpers.cs

```diff
@@ -275,6 +280,7 @@ public static async Task<bool> VerifyAndConfirmBranchExistsAsync(IRemote remote,
                 {
                     return PromptForYesNo("Continue?");
                 }
+                return false;
```

Comment:
This wasn't part of the issue but I think we should return false here if an exception was thrown and prompt == false. Right?

Reply:
I don't think so, we default to creating what you asked to be created if you pass --quiet, even if it's something that doesn't exist right now.

(I personally like that behavior, sometimes I create a subscription or create a default channel and afterwards push the branch)

Reply:
Ok. I'm going to revert that.